db/migrate/20240422150502_require_address1_and_city_and_phone_and_country_and_on_address.rb

```diff
@@ -0,0 +1,8 @@
+class RequireAddress1AndCityAndPhoneAndCountryAndOnAddress < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :spree_addresses, :address1, false
+    change_column_null :spree_addresses, :city, false
+    change_column_null :spree_addresses, :phone, false
```

Comment:
We don't actually have to enforce these three. If they are null, the records are invalid in Rails but we have some database violations. I ran this:

```
sql='select count(*) from spree_addresses where address1 is null;'
ansible all_prod -u openfoodnetwork -a "psql -h localhost openfoodnetwork ofn_user -c '$sql'"
```

Most databases are okay, but some are not:
```
openfoodnetwork.org.au | CHANGED | rc=0 >>
 count 
-------
    66

csicsoka.org | CHANGED | rc=0 >>
 count 
-------
  1236
```
Same numbers for `city`.

And for `phone` we have the same in AU but two other results:
```
openfoodnetwork.org.uk | CHANGED | rc=0 >>
 count 
-------
     4

csicsoka.org | CHANGED | rc=0 >>
 count 
-------
     0
```

Reply:
Mmh, Can/Should I remove this migration file now that this is merged ?

Reply:
I am looking at this, we are going to try fix the problematic data, but if it's too much of an issue we might revert some of the migrations